File: winshoes/Winshoes/Demos/CBuilder/SMTPClient/unitabout.cpp
//---------------------------------------------------------------------------
#include <vcl.h>
#pragma hdrstop

#include "unitabout.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TFormAbout *FormAbout;
//---------------------------------------------------------------------------
__fastcall TFormAbout::TFormAbout(TComponent* Owner)
    : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TFormAbout::ButtonCloseClick(TObject *Sender)
{
  Close();    
}
//---------------------------------------------------------------------------

File: winshoes/Winshoes/Demos/Delphi/SMTP/unitabout.pas
unit UnitAbout;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  ExtCtrls, StdCtrls;

type
  TFormAbout = class(TForm)
    Panel1: TPanel;
    Label1: TLabel;
    Label2: TLabel;
    Label3: TLabel;
    Label4: TLabel;
    ImageDPower: TImage;
    Label5: TLabel;
    Label6: TLabel;
    ButtonClose: TButton;
    procedure ButtonCloseClick(Sender: TObject);
    procedure FormClose(Sender: TObject; var Action: TCloseAction);
  private
  public
  end;

var
  FormAbout: TFormAbout;

implementation
{$R *.DFM}

procedure TFormAbout.ButtonCloseClick(Sender: TObject);
begin
  Close;
end;

procedure TFormAbout.FormClose(Sender: TObject; var Action: TCloseAction);
begin
  Action := caFree;
end;

end.
